internal/storage/image.go

```diff
@@ -298,8 +316,25 @@ func (svc *imageService) ImageStatus(systemContext *types.SystemContext, nameOrI
 		return nil, err
 	}
 
-	result := svc.buildImageResult(image, cacheItem)
-	return &result, nil
+	return svc.buildImageResult(systemContext, img, cacheItem, ref)
+}
+
+func inspectInfo(sc *types.SystemContext, ref types.ImageReference) (map[string]string, error) {
+	ctx := context.Background()
+	imgSrcRef, err := ref.NewImageSource(ctx, sc)
+	if err != nil {
+		return nil, errors.Wrap(err, "creating image source")
+	}
+	unparsed := image.UnparsedInstance(imgSrcRef, nil)
+	img, err := image.FromUnparsedImage(ctx, sc, unparsed)
```

Comment:
… or call `img, err := ref.NewImageSource(); defer img.Close()` and have it in one call instead of three, and don’t worry about calling `imgSrcRef`.

Reply:
Fixed via https://github.com/cri-o/cri-o/pull/3510#discussion_r402861078